### PR TITLE
Fix compile outputs to run on container

### PR DESCRIPTION
**Pull request by alexandrefreitasdasilva**
This PR fix the compile_outputs.py script to run on SIF.

### CODE REVIEW

**Review by Wilsonjsjunior — APPROVED** [review]
Show!

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name='ViralFlow',
-      version='1.0.1',
+      version='1.0.2',
```
Comment (dezordi) [inline setup.py:4]:
```suggestion
      version='1.1.0',
```

File: vfnext/main.nf [hunk 2]
```diff
@@ -42,7 +42,7 @@ ANSI_RESET = "\033[0m"
 
 log.info """
   ===========================================
-  VFNEXT v1.0.1
+  VFNEXT v1.0.2
```
Comment (dezordi) [inline vfnext/main.nf:45]:
```suggestion
  VFNEXT v1.1.0
```

File: wrapper/viralflow [hunk 3]
```diff
@@ -8,7 +8,7 @@ from sys import exit
 
 __author__ = "Antonio Marinho da Silva Neto"
 __license__ = "GPL"
-__version__ = "1.0.1"
+__version__ = "1.0.2"
```
Comment (dezordi) [inline wrapper/viralflow:11]:
```suggestion
__version__ = "1.1.0"
```

**Review by dezordi — APPROVED** [review]
Good job, as the -c 0.51 parameter modify the content of outputs, should increase one minor version on semantic release

**Pull request merged**